# Use EB modules for zlib and ncurses
# Author: Kurt Lust, UAntwerpen
# Developed from the 1.3.1 patch by Adam Huffman of The Francis Crick Institute
--- samtools-1.4-orig/Makefile	2017-03-16 15:11:35.236460000 +0100
+++ samtools-1.4/Makefile	2017-03-16 15:14:45.964895969 +0100
@@ -86,13 +86,13 @@
 
 ALL_CPPFLAGS = -I. $(HTSLIB_CPPFLAGS) $(CPPFLAGS)
 ALL_LDFLAGS  = $(HTSLIB_LDFLAGS) $(LDFLAGS)
-ALL_LIBS     = -lz $(LIBS)
+ALL_LIBS     = -L$(EBROOTZLIB) -lz $(LIBS)
 
 # Usually config.mk and config.h are generated by running configure
 # or config.status, but if those aren't used create defaults here.
 
 config.mk:
-	@sed -e '/^prefix/,/^LIBS/d;s/@Hsource@//;s/@Hinstall@/#/;s#@HTSDIR@#htslib-1.4#g;s/@HTSLIB_CPPFLAGS@/-I$$(HTSDIR)/g;s/@CURSES_LIB@/-lcurses/g' config.mk.in > $@
+	@sed -e '/^prefix/,/^LIBS/d;s/@Hsource@//;s/@Hinstall@/#/;s#@HTSDIR@#htslib-1.4#g;s/@HTSLIB_CPPFLAGS@/-I$$(HTSDIR)/g;s/@CURSES_LIB@/-L$$(EBROOTNCURSES)\/lib -lcurses/g' config.mk.in > $@
 
 config.h:
 	echo '/* Basic config.h generated by Makefile */' > $@
